### PR TITLE
Stop recommending the use of "doctrine:generate:entities"

### LINKED ISSUE #5070: Misuse of doctrine:generate:entities in documentation.
In the documentation, on the topic about generating getters and setters on entities, it's said to use doctrine:generate:entities MyEntity (http://symfony.com/doc/current/book/doctrine.html#generating-getters-and-setters)

However, according to an old issue on doctrine (https://github.com/doctrine/DoctrineBundle/issues/282), Ocramius answered (https://github.com/doctrine/DoctrineBundle/issues/282#issuecomment-56259385) that the goal of this command should not be used to generate getters and setters : 

Quote :
"@shideon yes, but we don't see real value in people running the generator over and over again. The generator is meant to be used to import legacy schemas, NOT to generate getters/setters and overwrite existing classes.

I realize that this is a crusade against a functionality that is useful to some people, but we really only had a lot of bugs and misuses coming from it, and that for customizations that should be applied post code-generation by the developer that is working on the project."

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #5070.

### CODE REVIEW

File: doctrine.rst [hunk 1]
```diff
@@ -400,56 +400,16 @@ Even though Doctrine now knows how to persist a ``Product`` object to the
...
-    $ php app/console doctrine:generate:entities AppBundle
+.. note::
 
-    # generates all entities of bundles in the Acme namespace
-    $ php app/console doctrine:generate:entities Acme
+    In the past, Symfony recommended the use of the ``doctrine:generate:entities``
+    command to let Doctrine generate these getter/setter methods. However, this
+    is now considered a bad practice. This command doesn't handle edge cases,
+    doesn't generate methods for associactions and creates unnecesary backups of
+    PHP classes.
```
Comment (xabbuh) [inline doctrine.rst]: I am not sure if we need to have this block.

**Review by xabbuh — APPROVED** [review]

**Review by oallain — APPROVED** [review]

File: doctrine.rst (hunk 1 above)
Comment (apfelbox) [inline doctrine.rst]: There also is a typo in "associactions"

Comment (javiereguiluz) [inline doctrine.rst]: I've removed it. Thanks!

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**